HACKING.md

```diff
@@ -153,13 +173,13 @@ files. These need to be rebuilt and published manually whenever they change.
 In order to publish new images, log in as `cukebot`. The password is in 1Password,
 in the "Cucumber Open Source" vault.
 
-   source ./scripts/functions.sh
-   # Standard image (for all builds except .NET)
-   docker_build Dockerfile
-   docker_push Dockerfile
-   # .NET image (for .NET builds only)
-   docker_build Dockerfile-dotnet
-   docker_push Dockerfile-dotnet
+     source ./scripts/functions.sh
```

Comment:
I think you changed from 3 space to 5 space indentation. I think 4 space is correct.